Formula/bullet.rb

```diff
@@ -43,15 +43,21 @@ def install
     args_static = args.dup << "-DBUILD_SHARED_LIBS=OFF"
     args_static << "-DBUILD_BULLET2_DEMOS=OFF" if build.without? "demo"
 
-    system "cmake", ".", *args_shared
-    system "make", "install"
+    mkdir "build" do
+      system "cmake", "..", *args_shared
+      system "make", "install"
 
-    system "make", "clean"
+      rm_rf Dir["*"]
 
-    system "cmake", ".", *args_static
-    system "make", "install"
+      system "cmake", "..", *args_static
+      system "make", "install"
 
-    prefix.install "examples" if build.with? "demo"
+      if build.with? "demo"
+        rm_rf Dir["examples/**/Makefile", "examples/**/*.cmake", "examples/**/CMakeFiles"]
+        prefix.install "examples"
```

Comment:
Can we dump these in `pkgshare` instead? 🙈

Reply:
Yep. Switched to `pkgshare`.